<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocaGUIClass</class>
 <widget class="QMainWindow" name="LocaGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>741</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1124</width>
    <height>741</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1124</width>
    <height>741</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LocaGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="analysisOptionGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1101</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis Options</string>
    </property>
    <widget class="QTabWidget" name="frequencyTabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>281</width>
       <height>156</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="addTRCButton">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>==&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeTRCButton">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;==</string>
     </property>
    </widget>
    <widget class="QListWidget" name="TRCListWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>212</width>
       <height>309</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="PROVListWidget">
     <property name="geometry">
      <rect>
       <x>878</x>
       <y>20</y>
       <width>212</width>
       <height>309</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="chosenTRClistWidget">
     <property name="geometry">
      <rect>
       <x>661</x>
       <y>20</y>
       <width>212</width>
       <height>309</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ERPcheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>eeg2erp ?</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>640</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process !</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="shellTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>1101</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="patientPathlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="locagui.qrc"/>
 </resources>
 <connections/>
</ui>
